v20.1/restore.md

```diff
@@ -34,6 +34,10 @@ You can restore:
 
 Because this process is designed for disaster recovery, a full cluster restore can only be run on a target cluster with no databases or tables.
 
+{{site.data.alerts.callout_info}}
+When you do a full cluster restore, it will restore the [enterprise license](enterprise-licensing.html) of the cluster you are restoring from. If you want to use a different license in the new cluster, make sure to [update the license](enterprise-licensing.html#set-a-license).
```

Comment:
nit: might help to clarify that that is update the license _after the restore is completed_